biolink-model.yaml

```diff
@@ -4211,6 +4327,32 @@ classes:
           - value: MONDO:0016419
             description: hereditary breast cancer
 
+
+  genotype to disease association:
+    is_a: association
+    comments:
+      - TODO decide no how to model pathogenicity
```

Comment:
@kshefchek Is this TODO still relevant?

Reply:
I think this just from copying over from variant to disease association, so the TODO might still stand

Reply:
👍